Header header
bosdyn_spot/Contact[] contacts